navigation/ca_collaborative_move_base/config/amcl_params.yaml

```diff
@@ -0,0 +1,34 @@
+use_map_topic: true
```

Comment:
This config file is not necessary since AMCL is already being executed somewhere else.